cognite/client/_api/time_series.py

```diff
@@ -249,7 +249,7 @@ def list(
             ).as_dicts()
 
         data_set_ids_processed = None
-        if data_set_ids or data_set_external_ids:
+        if data_set_ids is not None or data_set_external_ids is not None:
             data_set_ids_processed = IdentifierSequence.load(data_set_ids, data_set_external_ids).as_dicts()
```

Comment:
Sounds like a plan. Any thoughts on how we should pass variable names (for correct error messages)? I first saw `FunctionsAPI` catching and reraising with `function_id` instead of `id` (and same for XID)

Reply:
I pushed a suggestion now!

Reply:
...scope creep 🙈